actions/apt_action.go

```diff
@@ -49,6 +56,49 @@ func (apt *AptAction) Run(context *debos.DebosContext) error {
 	aptOptions = append(aptOptions, "install")
 	aptOptions = append(aptOptions, apt.Packages...)
 
+	if apt.LocalPackagesDir != "" {
+		localpackagesdir := path.Join(context.RecipeDir, apt.LocalPackagesDir)
+		destination, err := debos.RestrictedPath(context.Rootdir, "/usr/local/debs")
+		if err != nil {
+			return err
+		}
+		defer os.RemoveAll(destination)
```

Comment:
do the defers run at the end of the program or the end of the action? (my go is not good)
i think `destination` should be cleaned up after the action has completed. 
subsequent `apt` actions would also have this directory available, best to delete it after the action rather than after the recipe

Reply:
got it